backend/src/main/java/org/polsl/backend/controller/AffiliationController.java

```diff
@@ -35,8 +38,9 @@ public AffiliationController(AffiliationService affiliationService) {
    * @return lista przynależności
    */
   @GetMapping
-  public ResponseEntity<?> getAllAffiliations() {
-    return ResponseEntity.ok(affiliationService.getAllAffiliations());
+  public ResponseEntity<?> getAllAffiliations(@RequestParam(value="search", required=false) String search) {
+    Search<Affiliation> filtering = new Search<>(new Affiliation(), search);
+    return ResponseEntity.ok(affiliationService.getAllAffiliations(filtering.searchInitialization()));
```

Comment:
Myślałeś może o tym, żeby metoda `searchInitialization` była statyczna? Mogłoby nam być wygodniej.

Reply:
Myślałem, jak widać po historii commitów, ale metoda jest typu Specification<T> i jest napisane, że to T jest w metodzie statycznej niemile widziane.